### PR TITLE
audit-logging: fix proxy schema + cleanup package.json

### PR DESCRIPTION
**Pull request by sjvans**

### CODE REVIEW

File: samples/audit-logging/db/schema.cds [hunk 1]
```diff
@@ -1,4 +1,4 @@
 // Proxy to the base app's model to have ../srv/data-privacy.cds work in both
 // this extension app as well as when copied to incidents-app.
 // This file is not required for real projects.
-using from '@capire/incidents';
+using { sap.capire.incidents } from '@capire/incidents';
```
Comment (danjoa) [inline samples/audit-logging/db/schema.cds:4]: Should not be required, does it?

Comment (danjoa) [inline samples/audit-logging/db/schema.cds:4]:
```suggestion
using from '@capire/incidents';
```

**Review by danjoa — APPROVED** [review]
See inline comment

File: samples/audit-logging/db/schema.cds (hunk 1 above)
Comment (sjvans) [reply samples/audit-logging/db/schema.cds:4]: i could swear i got a compiler error this morning. but re-checked and, yes, not necessary.

**Pull request merged**